docs/tutorial.md

```diff
@@ -369,15 +431,21 @@ that by running `jj squash` with the `--interactive`/`-i` option on the
 third commit. Remember that `jj squash` moves all the changes from one commit
 into its parent. `jj squash -i` moves only part of the changes into its parent.
 Now try that:
+
 ```shell
 $ jj squash -i
-Using default editor 'meld'; you can change this by setting ui.diff-editor
-Working copy now at: 52a6c7fda1e3 ABCD
+Using default editor ':builtin'; you can change this by setting ui.diff-editor
+Working copy now at: mrxqplyk 52a6c7fd ABCD
+Parent commit      : kwtuwqnm 643061ac ABC
```

Comment:
This seems to be the only commit ID that wasn't present in the original output anywhere, so it can't be reconstructed. I've left it to how it was on my machine.

Reply:
Sounds good. I wouldn't have minded even if the commit IDs had all changed.